ocs_ci/helpers/helpers.py

```diff
@@ -3750,33 +3751,30 @@ def recover_mon_quorum(mon_pod_obj_list, mon_pod_running, ceph_mon_daemon_id):
         )
     running_mon_pod_yaml = dep_obj.get(resource_name=mon_deployment_name)
 
-    # Patch the mon Deployment to run a sleep
-    # instead of the ceph-mon command
+    # Patch the good mon Deployment so that deployment to stop the working of this mon without deleting the mon pod
     logger.info(
-        f"Edit mon {mon_deployment_name} deployment to run a sleep instead of the ceph-mon command"
+        f"Edit mon {mon_deployment_name} deployment to set 'initialDelaySeconds: 2000'"
```

Comment:
info statement doesn't match with the actual flow

Reply:
done